joomla/joomla-cms
# [com_fields] Add a parameter to trigger the additional plugin events in the news module

Pull Request for Issue #13699 .

### Summary of Changes
Adds a new parameter to the news module which allows to enable/disable the triggering of the additional plugin events onContentAfterTitle, onContentBeforeDisplay and onContentAfterDisplay.

### Testing Instructions
* Add some custom fieldsa to articles
* Create and publish a news module
* See that the custom fields appear
* Set the new parameter "Trigger Plugin Events" to "No" and verify that the fields no longer appear.

### Notes
Please have a look over the language strings. I'm not sure if they are correct (eg can the voting be shown there as well? The plugin event suggests so.) and proper english.

### Documentation Changes Required
None
